custom_components/waste_collection_schedule/waste_collection_schedule/source/sepan_remondis_pl.py

```diff
@@ -20,7 +20,9 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-API_URL = "https://sepan.remondis.pl/harmonogram"
+API_URL = "https://sepan.remondis.pl/harmonogram2024"
```

Comment:
I think it needs to end with `/`. Without that it returns 404.

Reply:
if you want to access it yes, but further down the code, it calls URL and adds /

"r = requests.get(f"{API_URL}/addresses/streets/{city_id}")"